DESIGN.md

```diff
@@ -0,0 +1,95 @@
+# Design Overview
+
+## Primary Tools
+
+**Frontend**: `Javascript`/`React`/`Webpack`/Airbnb-styleguide -- a standard frontend stack in keeping with Teleport's typical stack. For the sake of expediency I'll build for the latest stable Chrome release (and can therefore eschew `Babel` and a variety of other headaches) and will ignore mobile optimization.
```

Comment:
Scratch the remark about `Babel`, I was confused re: the need for a [loader](https://webpack.js.org/loaders/) for handling React-style js/jsx and so will be using `Babel` for that.